configuration/haproxy.go

```diff
@@ -8,4 +8,5 @@ type HAProxy struct {
 	ReloadCleanupCommand    string
 	ShutdownCommand         string
 	GraceSeconds            int
+	HostnameLabel           string
```

Comment:
add default setting in the two example configs https://github.com/strava/bamboo/blob/master/config/development.json